04_flora_cnell/_targets.R

```diff
@@ -0,0 +1,113 @@
+library(targets)
+
+options(tidyverse.quiet = TRUE)
+tar_option_set(packages = c('tidyverse', 'rnpn', 'terra', 'raster', 'sf','colorspace'))
+
+source("src/data_utils.R")
+source("src/plot_utils.R")
+
+proj <- "+proj=lcc +lat_1=30.7 +lat_2=29.3 +lat_0=28.5 +lon_0=-91.33333333333333 +x_0=999999.9999898402 +y_0=0 +ellps=GRS80 +datum=NAD83 +to_meter=0.3048006096012192 +no_defs"
+proj_aea <- '+proj=aea +lat_1=29.5 +lat_2=45.5 +lat_0=37.5 +lon_0=-96 +x_0=0 +y_0=0 +ellps=GRS80 +datum=NAD83 +units=m no_defs'
+
+list(
+  tar_target(
+    npn_layers, 
+    npn_get_layer_details()
+ ),
+ tar_target(
+    today,
+    Sys.Date()
+ ),
+ tar_target(
+   # most current daily anomaly
+   daily_anomaly,
+   get_npn_layer(layer_name = "Spring Indices, Daily Anomaly - Daily Spring Index Leaf Anomaly",
+                 npn_layers,
+                 layer_date = today),
+   format = 'file'
+  ),
+ tar_target(
+   # current date leaf out timing
+   daily_timing,
+   get_npn_layer(layer_name = "Spring Indices, Current Year - First Leaf - Spring Index",
+                 npn_layers,
+                 layer_date = today),
+   format = 'file'
+ ),
+ tar_target(
+   # historic date leaf out timing
+   hist_data,{
+     file_out <- 'out/si-x_30yr_avg_4k_leaf.tiff'
```

Comment:
Hitting an error here because there is no `out` directory. Typically I make the dir and drop in a `.empty` file so that the directory is committed and ready to go.